tests/drivers/dma/scatter_gather/src/test_dma_sg.c

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) 2021 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief Verify zephyr dma memory to memory transfer loops with scatter gather
+ * @details
+ * - Test Steps
+ *   -# Set dma configuration for scatter gather enable
+ *   -# Set direcation memory-to-memory
```

Comment:
```suggestion
 *   -# Set direction memory-to-memory
```
Just a nitpick. We have a bunch of little things like this in the codebase already, so it's not a huge deal.

Reply:
I wish I could change this and not dismiss the reviews, a downside of the new setup

Reply:
I'll reapprove if you push now- assuming @laurenmurphyx64 can provide second approval

Reply:
@danieldegrasse Yep :-)